CMakeLists.txt

```diff
@@ -7,11 +7,13 @@ project (fastq_to_fasta VERSION 1.0.0)
 
 ## BUILD
 
-# Make Release default build type
+# Make Release default build type. Coverage builds should use Debug.
 if (NOT CMAKE_BUILD_TYPE)
     set (CMAKE_BUILD_TYPE Release CACHE STRING
          "Choose the type of build, options are: Debug Release RelWithDebInfo"
          FORCE)
+elseif (SEQAN_APP_CODE_COVERAGE AND NOT CMAKE_BUILD_TYPE MATCHES "Debug")
```

Comment:
Seeing this, wouldn't be a `CMAKE_BUILD_TYPE = coverage` appropiate?

Reply:
I.e. adding custom build type and getting rid of the option?
Can check it out